src/pages/Profile/index.tsx

```diff
@@ -96,7 +97,7 @@ const Profile: FunctionComponent = () => {
 					</div>
 					<div className="profile__info">
 						<h2>{user.name || "Unnamed user"}</h2>
-						<p>{`${user.account.slice(0, 3)}...${user.account.slice(-4)}`}</p>
+						<p>{formatReadableAddress(user.account)}</p>
```

Comment:
Honestly, it would be better to create a separate PR for that instead of making one that big, but not a big deal